livedatapermission/README.MD

```diff
@@ -0,0 +1,66 @@
+# LiveData support
+Just in case of coroutine we saw above requesting permission is just a simple method call to [`PermissionManager`](livedatapermission/src/main/java/com/eazypermissions/livedatapermission/PermissionManager.kt) from your Activity/Fragment. It takes 3 parameters.
+1. An instance of AppCompactActivity or Fragment depending from where you are requesting permission.
+2. Request id.
+3. varargs of permission you want to request.
+
+This is how you request permissions from your Activity/Fragment.
+```kotlin
+PermissionManager.requestPermissions(
+                this,
+                REQUEST_ID,
+                Manifest.permission.ACCESS_FINE_LOCATION,
+                Manifest.permission.READ_CONTACTS
+            )
+```
+
+### Observing permission request result
+With just one simple step(implementing an interface) you are ready to observe the result of request.
+Your Activity/Fragment must implement `setupObserver` method of [`PermissionObserver`](https://github.com/sagar-viradiya/eazypermissions/blob/e1a36d5fb3ad487ac22da9b18e9b4c848cfcb74c/livedatapermission/src/main/java/com/eazypermissions/livedatapermission/PermissionManager.kt#L115) interface which expose LiveData<[`PermissionResult`](common/src/main/java/com/eazypermissions/common/model/PermissionResult.kt)>. Here is the definition of [`PermissionObserver`](https://github.com/sagar-viradiya/eazypermissions/blob/e1a36d5fb3ad487ac22da9b18e9b4c848cfcb74c/livedatapermission/src/main/java/com/eazypermissions/livedatapermission/PermissionManager.kt#L115)
```

Comment:
```
LiveData<[`PermissionResult`](common/src/main/java/com/eazypermissions/common/model/PermissionResult.kt)>
```
This relative path is no longer valid since this is not in top level README

try
`../common/src/main/java/com/eazypermissions/common/model/PermissionResult.kt`

Reply:
There are so many instances like this. Please make sure you change all and verify it is navigating properly.